packages/botonic-plugin-track-dynamo/tests/helpers/dynamo.ts

```diff
@@ -0,0 +1,23 @@
+import { ServiceConfigurationOptions } from 'aws-sdk/lib/service';
+import * as fs from 'fs';
+import * as os from 'os';
+
+/**
+ * At least Dynamo gets the keys from ~/.aws/credentials, but it does not get the region from ~/.aws/config
+ * @param conf
+ */
+export function testConfig(
+  conf?: ServiceConfigurationOptions
+): ServiceConfigurationOptions {
+  if (!conf || !conf.secretAccessKey || !conf.accessKeyId) {
+    let credentials = os.homedir() + '/.aws/credentials';
```

Comment:
Right now I don't know if in the lambda function ( is where this code will be executed when the bot is deployed ) we have in this route the  AWS credentials. Maybe we should use Environment Variables that can be stored in the lambda function.

Reply:
this is only for the unit tests. For production, the idea is injecting the keys from the bot definition through the plugin constructor options